YSE_App/models/frbtransient_models.py

```diff
@@ -173,6 +175,25 @@ def __str__(self):
     def natural_key(self):
         return self.name
 
+
+    def calc_DM_ISM(self):
```

Comment:
Not sure why this function is defined in the chime_kko.py file above as well. Duplicate left from an earlier commit?

Reply:
it isn't in `chime_kko.py`